src/main/java/com/cryptlex/lexfloatclient/LexFloatClient.java

```diff
@@ -56,6 +56,28 @@ public static void SetHostUrl(String hostUrl) throws LexFloatClientException {
         }
     }
 
+    /**
+     * Sets the permission flag. This function must be called on every start of your program 
+     * after SetHostProductId() function in case the application allows borrowing of licenses 
+     * or system wide activation.
+     *
+     * @param flag depending on your application's requirements, choose one of 
+     * the following values: LF_USER, LF_ALL_USERS.
+     * <ul>
+     * <li> LF_USER: This flag indicates that the application does not require admin or root permissions to run.</li>
+     * <li> LF_ALL_USERS: This flag is specifically designed for Windows and should be used for system-wide activations.</li>
+     * </ul>
+     * @throws LexFloatClientException
+     */
+    public static void SetPermissionFlag(int flag) throws LexFloatClientException {
+        int status;
+        status = Platform.isWindows() ? LexFloatClientNative.SetPermissionFlag(flag)
+                : LexFloatClientNative.SetPermissionFlag(flag);
```

Comment:
@muneebkq why is there a check for windows separately when we are just passing an int to it?